internal/cli/cmd/dac/build/build.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	outputFolderName = "built"
+	OutputFolderName = "built"
```

Comment:
I think this parameter should come from the config and that can be set during the command `setup` if you want to have a different default folder for the build.

And then you could still have a flag in the command `build` in case you want to define/override the default folder

Regarding the config, probably having a dedicated struct would be nice like:

```go
type DacConfig struct {
  Language        string `json: "language"`
  BuildFolderPath string `json: "build_folder_path"`
}

type Config struct {
	RestClientConfig perseshttp.RestConfigClient `json:"rest_client_config"`
	Project          string                      `json:"project,omitempty"`
	RefreshToken     string                      `json:"refresh_token,omitempty"`
        Dac              DacConfig                   `json:"dac"`
	filePath         string
	apiClient        api.ClientInterface
}
```

Reply:
I can help on this topic if you want

Reply:
As told offline imho this is kinda out of scope here, I'm planning to address this in a next PR.

Reply:
you could also put this PR on hold and come back on it when the folder is correctly managed. That would be the better way to go imh

Reply:
See https://github.com/perses/perses/pull/1753 ;)